protected/humhub/modules/user/models/ProfileField.php

```diff
@@ -310,7 +311,13 @@ public function checkType()
      */
     public function getUserValue(User $user, $raw = true): ?string
     {
-        return $this->fieldType->getUserValue($user, $raw);
+        $data = $this->fieldType->getUserValue($user, $raw);
+
+        if (!empty($this->fieldType->prefix)){
+            $data =Html::a($data, $this->fieldType->prefix.$data);
```

Comment:
Missing spaces it looks like between `prefix` and `.` as well as `$data` and `Html` and `=`